docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
Do we really need a .bat file?

Reply:
Perhaps not, but it's part of the standard Sphinx project template.  And given that we want to make devstack more Windows-friendly, it seems odd to remove it.